sdcm/cluster.py

```diff
@@ -4464,8 +4464,27 @@ def sct_dashboard_json_file(self):
                 sct_dashboard_json_filename = "scylla-dash-per-server-nemesis.master.json"
                 sct_dashboard_json_path = get_data_dir_path(sct_dashboard_json_filename)
             self._sct_dashboard_json_file = sct_dashboard_json_path
+            self.sct_dashboard_json_file_content_update(update_params=self.sct_dashboard_parameters_for_update(),
+                                                        json_file=self._sct_dashboard_json_file)
         return self._sct_dashboard_json_file
 
+    @staticmethod
+    def sct_dashboard_parameters_for_update():
+        update_params = {"$test_name": get_test_name()}
+        return update_params
+
+    @staticmethod
+    def sct_dashboard_json_file_content_update(update_params: dict, json_file: str):
+        # Read json data to the string
+        with open(json_file, 'r') as file:
+            json_data = json.dumps(json.load(file))
```

Comment:
what's happening here? why load then dump?

Reply:
`json.dumps` convert json object to string

Reply:
changed